docs/source/guides/federated_testnet_guide.md

```diff
@@ -0,0 +1,283 @@
+
+
+1. Install Python and Git
+    
+
+    If you don’t already have them, install Python and git.
+    As of January 2019, we are working with Python 3.6, 3.7, and 3.8. 
+
+
+    * Official Python Website: <https://www.python.org/downloads/> 
+    * Git Install Guide: <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>
+
+
+2.  Create Virtual Environment
+    
+    Create a system directory for the nucypher application code
+    
+    ```
+    $ mkdir nucypher
+    ```
+
+    Create a virtual environment for your node to run in using virtualenv
+    
+    ```bash
+    $ virtualenv nucypher -p python3
+    ...
+    ```
+    
+    Activate your virtual environment
+    
+    ```bash
+    $ source nucypher/bin/activate
+    ...
+    (nucypher)$
+    ```
+
+
+3. Install Nucypher
+    
+    Install nucypher with git and pip3 into your virtual environment
+    
+    ```bash
+    (nucypher)$ pip3 install git+https://github.com/nucypher/nucypher.git@master
+    ```
+    
+    Re-activate your environment after installing
+    
+    ```bash
+    $ source nucypher/bin/activate
+    ...
+    (nucypher)$ 
+    ```
+
+
+### Stage B | Configure Ursula
+
+
+1. Verify that the installation was successful
+    
+    Activate your virtual environment and run the nucypher --help command
+    
+    ```bash
+    $ source nucypher/bin/activate
+    ...
+    (nucypher)$ nucypher --help
+    ```
+    
+    You will see a list of possible usage options (--version, -v, --dev, etc.) and commands (accounts, configure, deploy, etc.). For example, you can use nucypher configure destroy to delete all files associated with the node.
+
+
+2. Configure a new Ursula node
+    
+    ```bash
+    (nucypher)$ nucypher ursula init --federated-only
+    ...
+    ```
+
+
+3. Enter your public-facing IPv4 address when prompted
+
+    `Enter Node's Public IPv4 Address: <YOUR NODE IP HERE>`
+
+
+4. Enter a password when prompted
+
+    `Enter a passphrase to encrypt your keyring: <YOUR PASSWORD HERE>`
+    
+    Save your password as you will need it to relaunch the node, and please note:
+    
+    - Minimum password length is 16 characters
+    - There is no password recovery process for NuFT nodes
+    - Do not use a password that you use anywhere else
+    - Your password may be displayed in logs or other recorded output.
+    - Security audits are ongoing on this codebase; for now, treat it as un-audited.
+
+## Running a NuFT Node
+
+### Stage C | Run the Node (Interactive Method)
+
+
+1. Connect to Testnet
+
+    NuCypher is maintaining a purpose-built endpoint to initially connect to the test network. To connect to the swarm run:
+    
+    ```bash
+    $(nucypher) nucypher ursula run --teacher-uri <SEEDNODE_URI>
+    ...
+    ```
+
+
+2. Verify Connection
+
+    This will drop your terminal session into the “Ursula Interactive Console” indicated by the `>>>`.  Verify that the node setup was successful by running the status command.
```

Comment:
```suggestion
    This will drop your terminal session into the “Ursula Interactive Console” indicated by the `>>>`.  Verify that the node setup was successful by running the `status` command.
```